rollup.node.js

```diff
@@ -1,24 +1,34 @@
-var fs = require("fs"),
-    rollup = require("rollup"),
-    dependencies = require("./package.json").dependencies;
+const rollup = require("rollup");
+const dependencies = require("./package.json").dependencies;
 
-rollup.rollup({
+// see below for details on the options
+const inputOptions = {
   input: "index.js",
   external: Object.keys(dependencies)
-}).then(function(bundle) {
-  return bundle.generate({
-    format: "cjs"
-  });
-}).then(function(bundle) {
-  var code = bundle.code;
```

Comment:
In the newer version of rollup, `bundle` is an object with member `output` of type Array. Looking at the new docs for rollup, I decided to go from their example:

https://rollupjs.org/guide/en/#rolluprollup